.github/workflows/e2e.yml

```diff
@@ -90,7 +90,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: public
-          key: public-cache-${{ hashFiles('pkg/webui/**') }}-${{ hashFiles('sdk/js/**/*.js', 'sdk/js/generated/*.json') }}-${{ hashFiles('config/webpack.config.babel.js') }}-${{ hashFiles('yarn.lock', 'sdk/js/yarn.lock')}}
```

Comment:
Would it make sense to move this step a little closer to where it's actually used? So above L145?

Reply:
Other than a cosmetic improvement it could save a couple seconds when an earlier step fails. :shrug:

Reply:
Yeah, it would indeed just be to keep related things together.